config/crd/bases/delivery.ocm.software_componentversions.yaml

```diff
@@ -126,8 +127,7 @@ spec:
                   type: object
                 type: array
               version:
-                description: Component versions refer to specific snapshots of a component.
-                  A common scenario being the release of a component.
+                description: Version specifies the version information for the ComponentVersion.
                 properties:
                   semver:
```

Comment:
The definition of version is varying across files & also in the same file:
https://github.com/open-component-model/ocm-controller/blob/main/config/crd/bases/delivery.ocm.software_localizations.yaml#L261